packages/synapse-react-client/src/components/EntityAclEditor/useNotifyNewACLUsers.test.ts

```diff
@@ -100,6 +104,7 @@ describe('useNotifyNewACLUsers', () => {
         principalId: mockUserData2.id,
         accessType: getAccessTypeFromPermissionLevel('CAN_EDIT_DELETE'),
       },
+      // removed user 3
```

Comment:
Modify this test to also verify that users _removed_ from an ACL are also not sent an email notification